modules/tinymce/src/core/main/ts/autocomplete/AutocompleteContext.ts

```diff
@@ -14,17 +14,18 @@ export interface AutocompleteContext {
 
 const stripTriggerChar = (text: string, triggerCh: string) => text.substring(triggerCh.length);
 
-const findChar = (text: string, index: number, ch: string): Optional<number> => {
+const findTrigger = (text: string, index: number, ch: string): Optional<number> => {
   // Identify the `char` in, and start the text from that point forward. If there is ever any whitespace, fail
   let i: number;
+  const str = text.substring(0, index);
 
   for (i = index - 1; i >= 0; i--) {
-    const char = text.charAt(i);
+    const char = str.charAt(i);
     if (isWhitespace(char)) {
       return Optional.none();
     }
 
-    if (char === ch) {
+    if (str.includes(ch, i)) {
```

Comment:
This should be using `Strings.contains` from Katamari instead.

Reply:
`Strings.contains` doesn't support necessary here `position` parameter

Reply:
Then maybe we should change it 😉